### ISSUE TITLE
Finalized form after resetting blank forms

### ISSUE THREAD

**Issue by dbemke** [getodk/collect#6496]
#### ODK Collect version
the store version 2024.2.4, the master version 8b72cd6469bed72f3c4d581718e14dba516a236c
#### Android version
10, 14
#### Device used
Redmi 9T, Pixel 7a
#### Problem description
If a user resets blank forms when there’s a finalized form, there are confusing pieces of information what happens to the form afterwards (described in steps below).
#### Steps to reproduce the problem
1. Finalize a form.
2. Go to Settings – Project Management- Reset and select Blank forms and reset them.
3. Go to Ready to send and open the form (there's information that it's a draft).
![finaliedDraft](https://github.com/user-attachments/assets/0297f849-654c-442b-8f48-97a74ffaf89e)
4. Send the form.
5. Go to "Sent" and check the subtext under the form ("Deleted form").
6. Reopen "sent" again and check the subtext ("Sent").
#### Expected behavior
In the information form step 3 it shouldn't be draft. After sending the subtext of the form in "Sent" should be "Sent"

**Comment by seadowg** [comment]
@dbemke I'm not able to reproduce this. Everything is the same until step 4: when I send the form it still gets submitted to the server (I can see the submission on Central).

**Comment by dbemke** [comment]
> it still gets submitted to the server (I can see the submission on Central).

Can it depend on the server? I tried twice on test.getodk.cloud and the submission was there, then I tried on staging and there weren't submissions

**Comment by dbemke** [comment]
Ok. I think I found it it's staging bug (probably).. the submission is there. I'll investigate it a bit more and edit the issue

**Comment by seadowg** [comment]
> Go to Ready to send and open the form (there's information that it's a draft).

Could you add a screenshot of this?

**Comment by seadowg** [comment]
Yeah, it looks like we don't have a separate message for viewing finalized forms in this case (opening a saved form where the form definition no longer exists). I do think we give the user a lot of chances to mess up here, and we've had a few bugs/discussions related to reset recently. I'm wondering if we should look to simplify the feature by either reducing the number of reset options (only allowing blank and saved forms to be reset together) or tweaking the options so it's "saved forms only" and "blank forms AND saved forms".

cc/ @alyblenkin
